forge/test/mlir/llama/tests/test_specific_ops_llama32.py

```diff
@@ -0,0 +1,480 @@
+# SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+#
+# SPDX-License-Identifier: Apache-2.0
+import os
+import pytest
+
+import torch
+from torch import nn
+
+import forge
+from forge.op.eval.common import compare_with_golden_pcc, compare_with_golden
+
+
+@pytest.mark.parametrize(
+    "shapes",
+    [
+        [(1, 11, 1), (1,)],
+        [(1, 32, 11, 64), (1, 32, 11, 64)],
+        [(1, 8, 11, 64), (1, 8, 11, 64)],
+        [(1, 32, 11, 11), (1, 1, 11, 11)],
+        [(1, 11, 2048), (1, 11, 2048)],
+    ],
+)
+def test_add(shapes):
+
+    if shapes[0] != shapes[1]:
+        pytest.xfail("eltwise_add broadcast not supported")
```

Comment:
For all xfail tests, can you create issues on MLIR within this milestone, and attach generated TTIRs as well? For easier tracking, let's also create reminder issues to remove those xfails once MLIR issues are fixed. Ofc, we can use the "Blocker" filed to track which issues from FFE is waiting some other issues on MLIR side :))

Let's create separate issues for each op type (e.g. add, cosine, etc).

_Note: This is not a blocker for merging this PR :))_

Reply:
Done!